olive/evaluator/olive_evaluator.py

```diff
@@ -861,7 +854,7 @@ def _evaluate_raw_latency(
     ):
         # pylint: disable=expression-not-assigned
         warmup_num, repeat_test_num, _ = get_latency_config_from_metric(metric)
-        session = model.prepare_session(inference_settings=self.get_inference_settings(metric), device=device)
+        session = model.prepare_session(inference_settings=None, device=device)
```

Comment:
Nit: add comments to tell now why None here? 
`None` as torch model have no inference settings.

Reply:
done